### DISCUSSION TITLE
Log UTC time

### DISCUSSION THREAD

**Discussion by Massi-X** [LSPosed/LSPosed#964] [Q&A] [answered]
Hi I see the logging is all done in UTC time. Is this done on purpose? It is a little bit misleading to see a time in the lsposed log and another in logcat.

**Comment by yujincheng08** [comment, marked as answer]
https://github.com/LSPosed/LSPosed/commit/70344ff9fd9ae9d64d5a4fb5d864269205893674

**Reply by Massi-X** [reply]
Great!
